cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -178,15 +173,12 @@ public void getInfo(HttpRequest request, HttpResponder responder,
                       @PathParam("namespace-id") String namespaceId,
                       @PathParam("stream") String stream) throws Exception {
     Id.Stream streamId = Id.Stream.from(namespaceId, stream);
+    checkStreamExists(streamId);
 
-    if (streamMetaStore.streamExists(streamId)) {
-      StreamConfig streamConfig = streamAdmin.getConfig(streamId);
-      StreamProperties streamProperties = new StreamProperties(streamConfig.getTTL(), streamConfig.getFormat(),
-                                                               streamConfig.getNotificationThresholdMB());
-      responder.sendJson(HttpResponseStatus.OK, streamProperties, StreamProperties.class, GSON);
-    } else {
-      responder.sendStatus(HttpResponseStatus.NOT_FOUND);
-    }
+    StreamConfig streamConfig = streamAdmin.getConfig(streamId);
```

Comment:
Maybe `streamAdmin.getConfg()` be throwing `NotFoundException` instead of having to remember to do `checkStreamExists` before every call.